### ISSUE TITLE
Scenarios for the use of s3im loss

### ISSUE THREAD

**Issue by aiyb1314** [Madaoer/S3IM-Neural-Fields#9]
When I use this loss on the MipNeRF360 dataset, I find that the quality of the rendered image deteriorates, is this not applicable to ubounded scenes? Can the same random ray be from different pixels in different training images?

**Comment by Madaoer** [comment]
@aiyb1314 
1. I think S3IM is a good choice for unbounded scenes, maybe you can finetune its weight. 
2. I don't understand this question. But I think it is so rare and will not change the results.

**Comment by aiyb1314** [comment]
The paper states that the light is sampled randomly, does it also mean that the same set of light can come from different images?

**Comment by Madaoer** [comment]
> The paper states that the light is sampled randomly, does it also mean that the same set of light can come from different images?

That's right, we sample them from different images.
